src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -154,6 +156,11 @@ export class RundownTimelineService implements RundownService {
     this.rundownEventEmitter.emitSetNextEvent(rundown)
     this.startAutoNext(timeline, rundownId)
 
+
+    if (rundown.getActiveSegment().definesShowStyleVariant) {
+      await this.ingestService.reloadIngestData(rundown.id)
+    }
+
     await this.deleteUnsyncedPreviousPart(rundown)
     await this.deleteUnsyncedSegments(rundown)
     await this.saveRundown(rundown)
```

Comment:
Perhaps we should start the reload flow after we are done with our own manipulations of the data and have saved that. If the reload flow is extremely fast then there is the option for a race condition here.

Reply:
I have moved it to be after the data manipulation, good point!